src/CMakeLists.txt

```diff
@@ -293,10 +293,14 @@ TARGET_LINK_LIBRARIES (FORTE_LITE ${LINK_LIBRARY})
 
 list(JOIN link_flags " " link_flags_string)
 
+if("${FORTE_ARCHITECTURE}" STREQUAL "Posix")
```

Comment:
This can be just `if (FORTE_ARCHITECTURE STREQUAL "Posix")`. But actually I think we don't need this guard at all. If people want to do this on win32 or other platforms, let them do so.

Reply:
I chose no guard at all.